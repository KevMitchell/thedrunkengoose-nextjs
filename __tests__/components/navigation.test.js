import { render } from '@testing-library/react'
import Navigation from '../../components/navigation'

test('button renders with correct label', () => {
  expect(render(<Navigation />)).toMatchSnapshot()
})
