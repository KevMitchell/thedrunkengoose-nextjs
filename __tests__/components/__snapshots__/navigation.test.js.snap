// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigation renders with all links 1`] = `
<div>
  <div
    class="jsx-443837931"
  >
    <img
      alt="The Drunken Goose"
      class="jsx-443837931 goose-title"
      src="./img/full-logo.png"
    />
    <a
      class="jsx-4272074681"
      href="/about"
    >
      About us
    </a>
    <a
      class="jsx-4272074681"
      href="/menu"
    >
      Menu
    </a>
  </div>
</div>
`;
